src/controllers/uploader.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response, NextFunction } from 'express'
+import aws from 'aws-sdk'
+
+import {
+  NotFoundError,
+  UnauthorizedError,
+  InternalServerError,
+  BadRequestError,
+} from '../helpers/apiError'
+aws.config.update({
+  region: 'us-east-1',
+  accessKeyId: process.env.AWSAccessKeyId,
+  secretAccessKey: process.env.AWSSecretKey,
+})
+const S3_BUCKET = process.env.bucket
+export const uploadImages = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const s3 = new aws.S3()
+    const fileName = req.body.fileName
+    const fileType = req.body.fileType
+    const s3Params = {
+      Bucket: S3_BUCKET,
+      Key: fileName,
+      Expires: 500,
```

Comment:
Do you know the unit here? Is it minutes?

Reply:
this is following from article, i really don't know :??

Reply:
Try to read the docs: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#getSignedUrl-property
This `signedUrl` will expire in 500 seconds